microsoft-365/admin/add-users/set-password-to-never-expire.md

```diff
@@ -27,7 +27,7 @@ description: "Learn how to set some individual user passwords to never expire, u
 
 ## Set the password expiration policy for your organization
 
-1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Settings</a> page.
+1. In the M365 Admin Center, go to the **Settings**, **Org settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Settings</a> page.
```

Comment:
Please replace step 1 with the following:

`1. In the Microsoft 365 admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Org settings</a> page.`